functionary/ui/templates/core/package_list.html

```diff
@@ -21,7 +21,11 @@
                                         {% endif %}
                                     </p>
                                 </a>
-                                {% if package.summary %}<p class="subtitle is-6">{{ package.summary }}</p>{% endif %}
+                                {% if package.summary %}
```

Comment:
Default to showing the summary if available, otherwise try show the description.